.prettierignore

```diff
@@ -0,0 +1,4 @@
+.cache
```

Comment:
Does this automatically ignore `node_modules`?

Reply:
I think so. but adding one line just to be sure